dist/parasails.js

```diff
@@ -340,7 +340,7 @@
             '<h1>Whoops</h1>'+
             '<p>'+
               '<span role="summary">An unexpected client-side error occurred.</span><br/>'+
-              '<pre>'+_.trunc(errorSummary, {length: 350})+'</pre>'+
+              _escape('<pre>'+_.trunc(errorSummary, {length: 350})+'</pre>')+
```

Comment:
@streleck this would make the `<pre>` tag not work -- I think you'll want to put the `_.escape()` directly around the result from `_.trunc()`